theano/sandbox/jaxify.py

```diff
@@ -623,7 +623,7 @@ def subtensor(x, *ilists):
 
 def jax_funcify_IncSubtensor(op):
 
-    idx_list = op.idx_list
+    idx_list = getattr(op, "idx_list", None)
```

Comment:
Nice that this is so simple.

Reply:
Yeah, the `Advanced*Subtensor1` `Op`s are just a specialization of the more general `Advanced*Subtensor` `Op`s to only one index parameter, so there's always a lot of overlap between them that can be leveraged.